### ISSUE TITLE
a simple but really important issue

### ISSUE THREAD

**Issue by dbsaronlau** [556556bt/HKCantonIME#7]
I try to type in Diu but apparently the word 屌 does not appear....
same situation in cantoneseinput.com; I guess it is a problem from the original word chart.

**Comment by 556556bt** [comment]
yes. an updated HKCanton IME db is working with a better sorting sequence. 
Please be patient.
